client/v2/algod/getLedgerStateDelta.go

```diff
@@ -8,15 +8,24 @@ import (
 	"github.com/algorand/go-algorand-sdk/v2/client/v2/common/models"
 )
 
+// GetLedgerStateDeltaParams contains all of the query parameters for url serialization.
+type GetLedgerStateDeltaParams struct {
+
+	// Format configures whether the response object is JSON or MessagePack encoded.
+	Format string `url:"format,omitempty"`
+}
+
 // GetLedgerStateDelta get ledger deltas for a round.
 type GetLedgerStateDelta struct {
 	c *Client
 
 	round uint64
+
+	p GetLedgerStateDeltaParams
 }
 
 // Do performs the HTTP request
 func (s *GetLedgerStateDelta) Do(ctx context.Context, headers ...*common.Header) (response models.LedgerStateDelta, err error) {
-	err = s.c.get(ctx, &response, fmt.Sprintf("/v2/deltas/%s", common.EscapeParams(s.round)...), nil, headers)
+	err = s.c.get(ctx, &response, fmt.Sprintf("/v2/deltas/%s", common.EscapeParams(s.round)...), s.p, headers)
```

Comment:
is there an endpoint to get the raw deltas?

Reply:
I'm confused by this response type, `response models.LedgerStateDelta`. I see that struct is deleted in this PR? and isn't `/v2/deltas/` in algod returning the raw state delta?